font/font.go

```diff
@@ -107,3 +110,36 @@ var JetBrainsMonoNLTTF []byte
 
 var JetBrainsMono string = base64.StdEncoding.EncodeToString(JetBrainsMonoTTF)
 var JetBrainsMonoNL string = base64.StdEncoding.EncodeToString(JetBrainsMonoNLTTF)
+
+var DefaultFontsDirectories []string
+
+func init() {
+	switch runtime.GOOS {
+	case "windows":
+		DefaultFontsDirectories = []string{filepath.Join(os.Getenv("windir"), "Fonts")}
+	case "linux":
```

Comment:
why not make this the default case then it should also be able to handle BSD systems too (at least in theory)

Reply:
if librsvg was installed, it won't meet this case.it easy for linux / BSD systems user to install librsvg

Reply:
I wonder if it would be better to do this at build time, rather than runtime? I.e. have `font_linux.go`, `font_darwin.go`, `font_windows.go` files, with the `//go:build` directives.